### ISSUE TITLE
Should we have a website for XVP?

### ISSUE THREAD

**Issue by webguyz** [xvpsource/xvp#1]
Colin Dean spent a lot of time and energy on his website. I would volunteer to host if it we can get the files from Colin. I believe he wants to step away totally since he no longer uses Xenserver.

**Comment by jonathanhoskin** [comment]
I was thinking that we take a snapshot of the current state of the xvpsource.org pages (using wget or similar), and migrate it into a Github Pages site. See this: http://pages.github.com

Ensures that no single entity has to take responsibility for ongoing hosting, and anyone can view the website source and submit changes through the standard Github mechanisms.

The existing xvpsource.org domain name could be CNAMEd to the Pages site, once it is stable: https://help.github.com/articles/setting-up-a-custom-domain-with-pages

Thoughts?

**Comment by shadowdao** [comment]
I like the idea of using github's pages so that no one party could hold the site hostage so to speak.  A lot of us in the community are or work for hosting companies, and while I would like to believe that everyone would do things in the best interest of the community and the project, I do not think we want to offer that temptation.

**Comment by jonathanhoskin** [comment]
Right, barring any other input, I'll go ahead and do a wget crawl of the existing site and generate a github page.

**Comment by jonathanhoskin** [comment]
From Colin:

> The current site is driven mostly by a single server-side PHP script: in case it's of any assistance, I've placed a copy of this on the site as index.php.txt so it can be downloaded as-is.

**Comment by bytestore** [comment]
Hi, you still need a website or hosting for your xvp project?
